### ISSUE TITLE
timestamp for err.log

### ISSUE THREAD

**Issue by Obscurator** [coreybutler/node-windows#259]
It would be great if the daemons xx.err.log would have the same timestamps like xx.wrapper.log

**Comment by mdodge-ecgrow** [comment]
Yes!  I was just looking at the error log and was frustrated because I had no idea when those errors occurred.

**Comment by Shriram-Nagarajan** [comment]
Had the same challenge while working on this. Created a log.js file as follows:

`const dateTimeMapper = function(formatArr){
	let transformedObj = {}; 
	for (let format of formatArr){
		transformedObj[format["type"]] = format["value"];
	} 
	return transformedObj;
}


const getLogPrefix = function(){
	let timeFields = ["year", "month", "day", "hour", "minute", "second", "dayPeriod"];
	let dateTimeFormat = new Intl.DateTimeFormat('en', { year: 'numeric', month: 'short', day: '2-digit' , hour : '2-digit', minute : '2-digit', second : '2-digit'});
	let arr = dateTimeFormat.formatToParts(new Date()).filter(format => ~timeFields.indexOf(format["type"]));
	let dateTimeMap = dateTimeMapper(arr);
	if(dateTimeMap){
		return dateTimeMap["year"]+"-"+dateTimeMap["month"]+"-"+dateTimeMap["day"]+" "+dateTimeMap["hour"]+":"+dateTimeMap["minute"]+":"+dateTimeMap["second"]+" "+dateTimeMap["dayPeriod"] + "    ";
	}
}

exports.log = {
    info : function(){
        arguments[0] = typeof arguments[0] === "string" ? getLogPrefix() + arguments[0] : arguments[0];
        console.log.apply(this, arguments);
    },
    warn : function(){
        arguments[0] = typeof arguments[0] === "string" ? getLogPrefix() + arguments[0] : arguments[0];
        console.warn.apply(this, arguments);
    },
    error : function(){
        arguments[0] = typeof arguments[0] === "string" ? getLogPrefix() + arguments[0] : arguments[0];
        console.error.apply(this, arguments);
    }
}`
